### ISSUE TITLE
Crash using HBQT and debugger

### ISSUE THREAD

**Issue by APerricone** [APerricone/harbourCodeExtension#41]
I have a similar problem.  I have one statement related to the QT add on in the HelloHarbour.prg. 

#include "hbqtgui.ch"

I added this statement to the program and then compiled and linked the program. When I start the program in debug mode, it stops and highlights line 1323 in the debugger.prg. When I then clicked on the Step Into button, the error message is thrown. I'll send the same to your email.

My program is stopping at line 1323 of the debugger program. Following is a snip of the code which is part of the case statement:

case HB_DBG_SHOWLINE

The line where the program stops starts here:

#ifndedf __ XHARBOURT__ //temp
    **if empty(tmp) .or. empt(t_oDebugInfo[..................**

When I click on the step into button, I get the following error message:

Error BASE/1070 Argument error: ==
Called from ERRORBLOCKCODE(1182)
Called from (b)__DBGENTRY(1334)
Called from __DBGENNTRY(1323)
Called from Main(6)

This fails before the first break point at the beginning of HelloHarbour.prg program. 

Please let me know if there's any additional info I can provide that will help resolve this issue.

Thanks for your help

_Originally posted by @dmwysocki61 in https://github.com/APerricone/harbourCodeExtension/issues/27#issuecomment-552879611_

**Comment by APerricone** [comment]
Because it is not relative to the other one, I created a new issue.
Can you write te output of harbour executable (without arguments, only first 2 lines)

**Comment by dmwysocki61** [comment]
hello, I am not sure what you need to see. I hope this helps


[
![Capture](https://user-images.githubusercontent.com/4809445/68995940-6e15e500-0861-11ea-828f-5f88edd2a23e.PNG)
](url)

**Comment by APerricone** [comment]
this is mine
![image](https://user-images.githubusercontent.com/22457168/68998470-f40e4c00-08b2-11ea-882f-890890d90575.png)

**Comment by dmwysocki61** [comment]
![Capture](https://user-images.githubusercontent.com/4809445/68998604-b5f93400-0882-11ea-8028-de521560f70c.PNG)

**Comment by dmwysocki61** [comment]
If I am using a version that is 3 years old, how do I get the most current version? I just downloaded from the Harbour download site and re-installed. I got the same version as I have been working with.

**Comment by APerricone** [comment]
it is not so much important, personally I download the source code with git then do make (win-make on windows).
I will check the [changeLog](https://raw.githubusercontent.com/harbour/core/master/ChangeLog.txt) to see if something between our versions can explain your problem.

**Comment by dmwysocki61** [comment]
Hi Antonino,

I resolved the problem. To resolve it, I did the following:

1. Created a VM Windows 10 on Azure to rule out any environmental issues.
2. Reinstalled Harbour by cloning the repository and compiling the source.
3. Installed VSCODE and all of the extensions per the instructions in Eric's article.
4. Up to this point, everything worked fine. The issue started when I downloaded and installed QTContrib into the Harbour folder. After this step, I started to experience the issue where the program was halted in the debugger program. When I click on the next step icon, the program crashes.
5. I restored the Harbour folder from backup. I then installed QTContrib into a separate folder called C:\QtContrib
6. I added a -L option to the hbp file to point to the folder where the required QT libs reside.

At this point the program is working. I still don't know how to add an option to the HBP file to include the *.hbc files. 

If anyone can help me with that, it would be greatly appreciated. So far, I've located the files and copied them to my working folder. So far, this is working but it would be much easier if I didn't have to worry about this step.

If anyone can help, it will be greatly appreciated.
